x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/index/IndexResolver.java

```diff
@@ -76,13 +76,12 @@
 public class IndexResolver {
 
     public enum IndexType {
-        STANDARD_INDEX("BASE TABLE", "INDEX"),
+        STANDARD_INDEX("TABLE", "INDEX"),
         ALIAS("VIEW", "ALIAS"),
-        FROZEN_INDEX("BASE TABLE", "FROZEN INDEX"),
+        FROZEN_INDEX("TABLE", "FROZEN INDEX"),
         // value for user types unrecognized
         UNKNOWN("UNKNOWN", "UNKNOWN");
 
-        public static final String SQL_BASE_TABLE = "BASE TABLE";
         public static final String SQL_TABLE = "TABLE";
```

Comment:
Since these constants are used somewhere else, it makes sense to move them outside `IndexType` and reuse them inside the enum declaration as well.